apps/omg_child_chain/lib/omg_child_chain/supervisor.ex

```diff
@@ -34,9 +34,10 @@ defmodule OMG.ChildChain.Supervisor do
     # prevent booting if contracts are not ready
     :ok = RootChain.contract_ready()
     {:ok, _contract_deployment_height} = RootChain.get_root_deployment_height()
+    fee_claimer_address = Application.fetch_env!(:omg, :fee_claimer_address)
```

Comment:
put the configuration fetcing into `OMG.Configuration`

```
def fee_claimer_address() do
    Application.fetch_env!(:omg, :fee_claimer_address)
  end
```
`OMG.Configuration.fee_claimer_address()`